config/locales/en.yml

```diff
@@ -91,6 +91,15 @@ en:
       male: "Male"
       female: "Female"
       other: "Other gender"
+    placement_type:
```

Comment:
Are these going to be constant?

Reply:
My thinking is no, they should be a resource on its own. But for the moment we have a hardcoded list of checkboxes for these, that's why they are translated here. I think this will evolve at some point and we'll change this too.